### ISSUE TITLE
Help Wanted

### ISSUE THREAD

**Issue by mwweb** [vdm-io/Joomla-Component-Builder#26]
When I took over the project that I am building out with Advanced Component Builder, I was come at the from the design side, having to rapidly transition into the development side. I will admit that on the development side, my knowledge is still relatively limited.

I want to get this component finished, and out to the community sooner than later. Is there anyone that you might recommend that is familiar with Advanced Component Builder that you might recommend I contact to see if they could help with some of the more advanced items (i.e. integrate drag/drop image upload and populate table, Google Maps, port gets from old version to this version, assist in front end site views/layouts/templates)?

**Comment by Llewellynvdm** [comment]
Hi, sorry for the delay... I was hopping someone else would respond to you :)

Okay so hear is the issue. Most who use the Component Builder at this time, are like you.. meaning they are using it to build their own projects and still figuring out how everything works. As you know the project is still very young and the community around it are mostly those who don't know how to build components, or how to write huge complex php projects.

But having said all that, hey those features you are looking for can be added with some custom scripting. I have build a component that actually use Google maps in it... here is a public use of the component called [group maps](http://www.tonata.org/map)

I can give you the component... we may need to have a skype call to sort that out... let me know.

Helping you with the rest of your ideas... hmmm my time is very limited... but we can see what I can do for you. Let me know when we can have a chat :)

`

**Comment by mwweb** [comment]
Sorry for the delayed response. I have been busy building out some native Android and iOS apps for my clients, that pull data via a custom API from the old version 1 of the component I have. Now that I have roughly 95% of the work done on that, I have a little more time available.

Below is the link to the article I wrote about Advanced Component Builder. I submitted this to the Joomla! Community Magazine, and I hope that they will actually publish it as well.

[http://mostwantedwebservices.com/hosting-and-design-blog/entry/joomla-extension-building-options](url)
